client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -1100,6 +1137,11 @@ export default {
       this.refresh(true)
     },
 
+    showMoreRecords () {
```

Comment:
can we call this something else. Show more records doesn't really describe it well.
For example `handlePerPageChange` would be more descriptive.